static/sass/config/_scale.scss

```diff
@@ -84,9 +83,27 @@ $pattern-sizes: (
 /// @link http://oddbird.net/accoutrement-scale/sassdoc/
 ///   Accoutrement Scale
 $page-sizes: (
-  'small-page': 20rem,
-  'full-content': 40rem,
-  'blockquote': 55rem,
+  'small-page': 34rem,
+  'medium-page': 55rem,
+  'large-page': 75rem,
```

Comment:
I'm curious how these sizes are meant to relate to the sizes below. Maybe just needs a bit more documentation?